app/javascript/src/components/Common/TitleBar/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { Search, Plus } from "@bigbinary/neeto-icons";
+import { Input, Button } from "@bigbinary/neetoui/v2";
+import { Header } from "@bigbinary/neetoui/v2/layouts";
+
+const TitleBar = ({
+  children,
+  title,
+  toggleMenu,
+  buttonLabel,
+  onButtonClick,
+}) => (
+  <Header
+    menuBarToggle={toggleMenu}
+    actionBlock={
+      <div className="flex flex-row">
+        {children}
+        <Input
+          prefix={<Search />}
+          placeholder="Search for something"
+          className="mx-3"
+        />
+        <Button
+          label={buttonLabel}
+          icon={Plus}
+          onClick={onButtonClick}
+          style="primary"
+        />
+      </div>
+    }
+    title={title}
+  />
+);
+
+export default TitleBar;
```

Comment:
We don't need to do `Common/TitleBar/index.jsx`. This could simply be `Common/TitleBar.jsx`. 
Also, I don't see a specific reason why we should be extracting neetoUI `Header` component into another reusable component.

Reply:
`actionblock` has the same UI for all title bars, This was done to avoid repeating that part.